cdi/common-jakarta/pom.xml

```diff
@@ -14,6 +14,12 @@
     <artifactId>infinispan-cdi-common-jakarta</artifactId>
     <name>Infinispan CDI Jakarta EE support common</name>
 
+    <properties>
+        <maven.javadoc.skip>true</maven.javadoc.skip>
```

Comment:
by skipping won't make ossrh to complain?

Reply:
No, because it is being attached by the transformer plugi